org.kde.konquest.json

```diff
@@ -0,0 +1,46 @@
+{
+    "id": "org.kde.konquest",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.15",
+    "sdk": "org.kde.Sdk",
+    "command": "konquest",
+    "rename-icon": "konquest",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=x11",
```

Comment:
```suggestion
        "--socket=fallback-x11",
```

Reply:
Generally the KDE apps have been more "wayland available" than "wayland first" up until now.

Reply:
Let's keep it that way for uniformity but my understanding is that KDE apps run natively on Wayland also under GNOME on latest runtime so that's something that should be addressed at some point.